main/src/com/google/refine/grel/ast/OperatorCallExpr.java

```diff
@@ -101,6 +102,7 @@ public Object evaluate(Properties bindings) {
                     } else if ("*".equals(_op)) {
                         return n1 * n2;
                     } else if ("/".equals(_op)) {
+                        if(n2 == 0&& n1 == 0) { return Double.NaN; }
```

Comment:
```suggestion
                        if (n2 == 0 && n1 == 0) {
                            return Double.NaN;
                        }
```

Reply:
We normally have linting checks which should pick this up but it looks like they are not quite strict enough. You could do a similar change above.

Reply:
Alright, I will. Thanks! :)